### ISSUE TITLE
The command api:routes only shows Clouse under the Action column [Again]

### ISSUE THREAD

**Issue by netojose** [dingo/api#1169]
When using simple form router registration:
$api->get('foo/bar', 'FooController@fooAction');
It's ok.

But, when in a complex scenario, like this:

$api->get('foo/bar', ['middleware' => 'api.auth'], 'FooController@fooAction');
$api->get('foo/bar', ['middleware' => 'api.auth', uses => 'FooController@fooAction']);

The command api:routes only shows Clouse under the Action column again :/ .

**Comment by farhanwazir** [comment]
In your senario, routes group will be efficient and easy manageable. 
```php
$api->group (['middleware' => 'api.auth'], function($api){
//your routes
});
```

**Comment by netojose** [comment]
> In your senario, routes group will be efficient and easy manageable.
> 
> ```
> $api->group (['middleware' => 'api.auth'], function($api){
> //your routes
> });
> ```

But if I want to do only once, I need to create a group to put only one route inside a group?

**Comment by specialtactics** [comment]
Yes, a lot of route commands do not support closures - and it is actually like that even in vanilla Laravel.

Closures are not really supposed to be used in the routes file, the feature is there for very quick prototyping, but you should never write your application like that.

**Comment by netojose** [comment]
But I didn't use closures. What I said, was the dingo API show "closure" string instead of show the correct information when using route groups.

**Comment by specialtactics** [comment]
@netojose Can you post up the output of your api:routes? I don't believe I've seen what you are referring to (I think)

**Comment by netojose** [comment]
@specialtactics , I haven't the project on the my machine anymore (I opened this ticket on 25 Aug 2016). Tomorrow I will look at my repositories, or create a new one and try to reproduce the error.

Thanks \o/

**Comment by specialtactics** [comment]
No worries.

If you no longer care about the issue I can close the ticket as well. Maybe it was fixed along ago :smile:
